pegasus_wrapper/resources/sites.xml

```diff
@@ -26,7 +26,7 @@
         <profile namespace="pegasus" key="style">glite</profile>
 
         <!--  batch queue to submit jobs to - both are required -->
-        <profile namespace="pegasus" key="queue">gaia</profile>
```

Comment:
Why this change?

Reply:
@LichMaster98 might have a better idea on this one

Reply:
This change can be undone, this modification was made when investigating why jobs from the test workflow were always being submitted to the `gaia` partition rather than scavenge.

Reply:
If it works with this omitted entirely, I suggest deleting the line below rather than commenting it out.